data/src/main/java/dev/baseio/discordjetpackcompose/models/Country.kt

```diff
@@ -0,0 +1,19 @@
+package dev.baseio.discordjetpackcompose.models
+
+
+import androidx.annotation.Keep
+import com.google.gson.annotations.SerializedName
+
+@Keep
+data class Country(
```

Comment:
@shubhamsinghshubham777 also let's move the logic to fetch countries in data module

Reply:
Do you mean we need to create a usecase for fetching countries?